feat(changelog): postgresql-and-mysql-added-ability-to-remove-public-end 2024-03-25 (#2957)
scaleway/docs-content: changelog/march2024/2024-03-25-postgresql-and-mysql-added-ability-to-remove-public-end.mdx

---

> title: Ability to remove public endpoints from Console

Reviewer: ```suggestion
title: Ability to remove public endpoints from the console
```

> title: Ability to remove public endpoints from the console

Reviewer 2: ```suggestion
title: You can now remove public endpoints from the Scaleway console
```

status: added
author:
    fullname: 'Join the #database channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-03-25
category: databases
product: postgresql-and-mysql
---

> The Network management on Managed DB for PostgreSQL and MySQL has been slightly revamped and you can now add/remove public endpoints directly from your Console, may it be those attached to Database Instances or Read-Replicas.

Reviewer: ```suggestion
Network management on Managed DB for PostgreSQL and MySQL has been improved, and you can now add/remove public endpoints directly from the Scaleway console. This includes endpoints attached to both Database Instances and Read-Replicas.
```

> Network management on Managed DB for PostgreSQL and MySQL has been improved, and you can now add/remove public endpoints directly from the Scaleway console. This includes endpoints attached to both Database Instances and Read-Replicas.

Reviewer 2: ```suggestion
Network management on Managed Databases for PostgreSQL and MySQL has been improved! You can now add/remove public endpoints directly from the [Scaleway console](https://console.scaleway.com). This includes endpoints attached to both Database Instances and Read-Replicas.
```

> Refer to [this documentation](https://www.scaleway.com/en/docs/managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint/) for more information.

Reviewer: ```suggestion
Refer to [our documentation](https://www.scaleway.com/en/docs/managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint/) for more information.
```

> Refer to [our documentation](https://www.scaleway.com/en/docs/managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint/) for more information.

Reviewer 2: ```suggestion
Refer to [our documentation](/managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint/) for more information.
```
